templates/trivy/trivy-sts.yaml

```
@@ -105,3 +106,4 @@ spec:
         resources:
           requests:
             storage: {{ $trivyPVC.size | quote }}
+{{ end }}
```

We can trim left whitespaces here:

```suggestion
{{ end }}
{{- end }}
```